app/styles/tasks.css

```diff
@@ -36,49 +41,107 @@
 
 /* progress bar */
 
+.wrapper-div{
+  margin: 1em;
+  padding: 1rem;
+}
+
 .progress-bar {
-  background-color: #BFADA3;
-  height: 15px;
-  border-radius: 20px;
+  background-color: #fff;
+  height: 40px;
+  border-radius: 25px;
   overflow: hidden;
-  box-shadow: 2px 0 10px inset rgba(0,0,0,0.2);
   margin-top: 16px;
   position: relative;
 }
 
-.percent-20 {
-  width: 20%;
+.progress-bar-container{
+  padding: 1em;
 }
 
-.percent-40 {
-  width: 40%;
+input[type=range] {
+  -webkit-appearance: none;
+  width: 80%;
 }
 
-.percent-60 {
-  width: 60%;
+input[type=range]::-webkit-slider-runnable-track {
+  width: 50%;
+  height: 5px;
+  cursor: pointer;
+  background-color: var(--slider-background);
+  box-shadow: 0px 0px 0px #000000;
+  background: var(--slider-colour);
+  border-radius: 1px;
+  border: 0px solid #000000;
+
+}
+input[type=range]::-webkit-slider-thumb {
+  box-shadow: 0px 0px 0px #000000;
+  border: 1px solid #3724e3;
+  height: 18px;
+  width: 18px;
+  border-radius: 50%;
+  background: #A1D0FF;
+  cursor: pointer;
+  -webkit-appearance: none;
+  margin-top: -7px;
 }
-
-.percent-80 {
-  width: 80%;
+input[type=range]:focus::-webkit-slider-runnable-track {
+  background: var(--slider-colour);
 }
-
-.bar {
-  height: 100%;
-  background-color: #041484;
-  background-image: linear-gradient(
-    -45deg,
-    rgba(255, 255, 255, .2) 25%,
-    transparent 25%,
-    transparent 50%,
-    rgba(255, 255, 255, .2) 50%,
-    rgba(255, 255, 255, .2) 75%,
-    transparent 75%,
-    transparent
-  );
-  background-size: 30px 30px;
-  animation: move 2s linear infinite;
-  box-shadow: 2px 0 10px inset rgba(0,0,0,0.2);
-  transition: width 2s ease-out;
+input[type=range]::-moz-range-track {
```

Comment:
Eager to see what each css properties does here

Reply:
https://css-tricks.com/styling-cross-browser-compatible-range-inputs-css/